### ISSUE TITLE
long description does not render in reST on pypi

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/pmda#40]
https://pypi.org/project/pmda/ does not render nicely in the 0.1.0 release. The content is properly declared but [checking the README rendering](https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi) with [readme_renderer](https://github.com/pypa/readme_renderer) 
```bash
python setup.py check -r -s
/nfs/homes2/oliver/Library/miniconda2/envs/mda3/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
  warnings.warn(msg)
running check
warning: Check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too

warning: Check: The project's long_description has invalid markup which will not be rendered on PyPI. The following syntax errors were detected:
line 40: Error: Unknown interpreted text role "class".


error: Please correct your package.
```
showed that we are using Sphinx constructs that the PyPi renderer does not like. 

(The reason for these constructs is that we are using the README also as the [landing page for the PMDA docs](https://www.mdanalysis.org/pmda/).)

**Comment by orbeckst** [comment]
@kain88-de the meta data should contain an email address for you:
```
warning: Check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too
```
Which one should go in there?

**Comment by kain88-de** [comment]
The google address would be good

**Comment by orbeckst** [comment]
Can you add it yourself, please?

I'm currently away from a real computer. 

--
Oliver Beckstein
email: orbeckst@gmail.com

**Comment by kain88-de** [comment]
done
